README.md

```diff
@@ -49,9 +49,10 @@ def foo(request):
 # equals to
 def foo(request):
     bar = Bar.object.all()
-    return render_to_response('template.html',
-                                {'bar': bar},
-                                context_instance=RequestContext(request))
+    return render(request,
+                  'template.html',
+                  {'bar': bar},
+                  context_instance=RequestContext(request))
```

Comment:
`render` does not take a `context_instance` variable, it can safely be omitted here.

Reply:
Ouch! Good catch. So unlucky that tests are passed.

Reply:
Addressed in 98376144ae390e23b0e6ebee02b231b74a36dd0e